### PR TITLE
WIP Fix storybook integration and add a default story with basic schema

### PR DESCRIPTION
**Pull request by danleyb2**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `develop` branch (or to a previous version branch), _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: src/components/form-generator.vue [hunk 1]
```diff
@@ -59,12 +59,13 @@ export default {
     return {}
   },
   created: function() {
-    this.$watch(
```
Comment (danleyb2) [inline src/components/form-generator.vue]: @aldarund this kept throwing errors so i had to comment it out to test.

Please help fix it on the storybook

File: package.json [hunk 2]
```diff
@@ -108,8 +108,8 @@
     "terser": "^3.10.1",
     "vue-eslint-parser": "^3.2.2",
     "vue-jest": "^3.0.0",
-    "vue-loader": "^15.4.2",
-    "vue-template-compiler": "^2.5.17"
+    "vue-loader": "^13.5.2",
```
Comment (aldarund) [inline package.json]: why downgrade? it will cause problems when used in main project

Comment (danleyb2) [reply package.json]: `start-storybook` doesn't work with ^15 
an error about missing loaders. 
could you please test and advice on how to resolve.

**Review by danleyb2 — COMMENTED** [review]
i didn't **Submit Review**

File: package.json (hunk 2 above)
Comment (aldarund) [reply package.json]: it should work with 15. I didnt really used storyblock but I suppose you need to modify webpack config to add vue storyblock. Something like this

  config.module.rules.push({
    resourceQuery: /blockType=story/,
    loader: 'vue-storybook',
  })

Comment (aldarund) [reply package.json]: this whole stotyblcok is really dated and probably need to be updated to be properly used. Its from template and was never sued

File: src/components/form-generator.vue (hunk 1 above)
Comment (aldarund) [reply src/components/form-generator.vue]: what error? Like i said i didnt really used storybook...

Comment (danleyb2) [reply src/components/form-generator.vue]: @aldarund  Here is the error message from browser console 
It really not but. I am just not sure on how to add the `_this.$invalid` function 


```
Vue warn]: Error in getter for watcher "function () {
      return _this.$invalid();
    }": "TypeError: _this.$invalid is not a function"

found in

---> <VuetifyFormGenerator> at src/components/form-generator.vue
       <Anonymous>
         <VApp>
           <Anonymous>
             <Root>
```

File: package.json (hunk 2 above)
Comment (danleyb2) [reply package.json]: Okay i have resolved the issue but had to upgrade and add more Dev dependencies. this is outdated

**Pull request merged**